BOJ2439.java

```diff
@@ -0,0 +1,26 @@
+package assignment;
+
+import java.io.*;
+
+public class BOJ2439 {
+    public static void main(String[] args) throws IOException {
+        // TODO Auto-generated method stub
+        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
+
+        int n = Integer.parseInt(br.readLine());
+
+        for(int i=n;i>0;i--){
```

Comment:
이 구조는 동작은 맞지만 공백과 별의 개수를 직관적으로 파악하기 어렵네요 😢
`i=1; i<=n; i++` 이런식으로 작성하면 줄 번호가 직관적이라 가독성이 더 좋을 것 같아요-!

```suggestion
        for(int i=i; i<= n; i++){
```

이런 식으로 내부 for문도 수정하면 좋을 것 같습니다~